roles/nginx/tasks/ssl.yml

```diff
@@ -28,10 +25,21 @@
     state: stopped
   when: not fullchain.stat.exists
 
-- name: generate ssl certificates for {{ domain }}
-  raw: certbot certonly --standalone --email {{ email }}
-       --agree-tos -n -d {{ domain }},www.{{ domain }}
-  when: not fullchain.stat.exists
+
+- name: "generate staging ssl certificates for {{ domain }}"
+  command: "certbot certonly --standalone --email {{ email }}
+       --agree-tos -n -d {{ domain }},www.{{ domain }} --staging"
+  when:
+    - not fullchain.stat.exists
+    - ssl == 'staging'
```

Comment:
I'd rather keep this in line with the other generation, and use the jinja templating to do `{% if ssl_use_staging %} --staging{% endif %}` vs having two separate tasks paths that do, effectively, the same thing.

You may notice the variable name change too, I don't like that we are normally using a boolean for `ssl` and then checking a string here. I think it may be cleaner to supplement `ssl` with something more descriptive as `ssl_use_staging` or similar.